### PR TITLE
Fixed nav gradient to match UX design.

### PR DESCRIPTION
**Pull request by jdhodges**
#638. Replaces #639.

### CODE REVIEW

**Comment by jleft** [comment]
(Closing and reopening; trying to trigger Travis build.)

File: src/assets/js/chart/nav.js [hunk 1]
```diff
@@ -85,16 +87,29 @@ export default function() {
 
     var brushMask = fc.series.area()
         .xValue(function(d) { return d.date; })
-        .yValue(function(d) { return d.close; })
+        .y1Value(function(d) { return d.close; })
+        .y0Value(function(d) { return yScale.domain()[0]; })
+        .decorate(function(selection) {
+            selection.enter().attr('fill', 'url("#brush-gradient")');
+        })
         .xScale(xScale)
```
Comment (jleft) [inline src/assets/js/chart/nav.js]: No longer needed? As used in multi.

File: src/assets/js/chart/nav.js [hunk 2]
```diff
@@ -85,16 +87,29 @@ export default function() {
...
         .yScale(yScale);
```
Comment (jleft) [inline src/assets/js/chart/nav.js]: No longer needed? As used in multi.

**Comment by jleft** [comment]
In #639, the [brush gradient was removed from the extent](https://github.com/ScottLogic/BitFlux/pull/639/files?diff=unified#diff-d28ca23127750614efe6eedcddd205eeL59), should this be done here too?

Just wondering if there was a reason open a new PR for this, rather than update the previous one (#639)?

**Comment by jdhodges** [comment]
I tried to trigger the travis build by closing and reopening but didn't have the option to reopen.

**Comment by jleft** [comment]
> I tried to trigger the travis build by closing and reopening but didn't have the option to reopen.

Ah okay, no problem - in future, if that happens again, please can you just let me know and I can reopen? Makes it a lot easier to review the changes if they're in a single PR, rather than multiple :smiley:

**Comment by jdhodges** [comment]
Yeah will do, had a bit of a nightmare with github today.

**Comment by jleft** [comment]
> Yeah will do, had a bit of a nightmare with github today.

Thanks, no worries :)

LGTM :+1:

**Comment by jdhodges** [comment]
image

Houston, we have a problem.

**Comment by jleft** [comment]
Mask was removed instead of gradient, I guess is causing the problem?

https://github.com/ScottLogic/BitFlux/pull/724/files#diff-d28ca23127750614efe6eedcddd205eeL59

**Comment by jdhodges** [comment]
Yeah that was it. I think it's time for me to go home haha.

**Pull request merged**